docs/api-reference/carto/carto-layer.md

```diff
@@ -107,23 +77,21 @@ Required. Data type. Possible values are:
 
 ##### `connection` (String)
 
-Required when `apiVersion` is `API_VERSIONS.V3`.
-
-Name of the connection registered in the CARTO workspace.
+Required. Name of the connection registered in the CARTO workspace.
```

Comment:
Not sure about this. This property is optional. Someone can come here and believe they need to use it for CARTO 2. I agree with removing specific things but this is the official docs and we shouldn't say a prop is mandatory if it is not.